### DISCUSSION TITLE
how to start with no password on server

### DISCUSSION THREAD

**Discussion by ghost** [mbround18/valheim-docker#156] [Q&A] [answered]
as far as i can tell the environment flag for password HAS to be set. is there a way to start a true public? no pass? i tried removing the password out of the environment area of the docker compose file but the game still wanted a password

**Comment by mbround18** [comment]
As of right now, this is not yet supported by the Valheim developers. You can use VelheimPlus to disable the server password but that feature is not quite merged into the latest release.

**Comment by AtroposOrbis** [comment, marked as answer]
> as far as i can tell the environment flag for password HAS to be set. is there a way to start a true public? no pass? i tried removing the password out of the environment area of the docker compose file but the game still wanted a password

You can now do this using Longship as well. Just leave the password blank in the longship.cfg after running the server with longship installed.
